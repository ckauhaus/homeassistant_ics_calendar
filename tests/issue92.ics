BEGIN:VCALENDAR
VERSION:2.0
PRODID:Microsoft Exchange Server 2010
BEGIN:VEVENT
UID:646343bb0d294
DTSTART;VALUE=DATE:20221231
SEQUENCE:0
TRANSP:OPAQUE
SUMMARY:Altpapier
CLASS:PUBLIC
X-MICROSOFT-CDO-ALLDAYEVENT:TRUE
DTSTAMP:20230516T085003Z
END:VEVENT
END:VCALENDAR