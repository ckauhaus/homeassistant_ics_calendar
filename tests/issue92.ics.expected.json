[
    {
        "all_day": true,
        "start": "2022-12-31",
        "end": "2023-01-01"
    }
]